client/clientauth.js

```diff
@@ -116,7 +126,7 @@ module.exports = ( fastify, opts, done ) => {
 			method: "POST",
 			host: server.ip,
 			port: server.authPort,
-			path: `/authenticate_incoming_player?id=${request.query.id}&authToken=${authToken}&serverAuthToken=${server.serverAuthToken}`
+			path: `/authenticate_incoming_player?id=${request.query.id}&username=${account.username}&authToken=${authToken}&serverAuthToken=${server.serverAuthToken}`
```

Comment:
Wait, with API change, does that mean old clients won't be able to register anymore? If so, we might wanna adjust it to allow clients to use both versions, move update client to use new version in next release, and then deprecate the old one once a major release (like `2.0`) hits.

Reply:
Resolved in discussion on Discord. API for game client doesn't change.